dev/io.openliberty.microprofile.telemetry.1.0.internal.jaeger_fat/fat/src/io/openliberty/microprofile/telemetry/internal/tests/ZipkinOtelCollectorTest.java

```diff
@@ -67,6 +70,9 @@ public class ZipkinOtelCollectorTest {
     @ClassRule
     public static Network network = Network.newNetwork();
 
+    @ClassRule
+    public static RepeatTests r = MicroProfileActions.repeat("spanTestServer", MicroProfileActions.MP60, MicroProfileActions.MP61);
+
     private static final Class<?> c = ZipkinTest.class;
 
     public static final int OTLP_GRPC_PORT = 4317;
```

Comment:
is this port number always fixed and not configurable/variable?

Reply:
Yes it's the fixed port that the backend exporter receives spans from